.circleci/config.yml

```diff
@@ -179,6 +179,7 @@ jobs:
   docker-build-and-publish: &docker-job
     docker: &docker-image
       - image: cimg/node:16.15
+    resource_class: xlarge
```

Comment:
Should these unrelated changes be in a separate PR?

Reply:
Since the containers are not build (yet) on PRs, there is no way to test them on CI atm. So i thought its fine...